### ISSUE TITLE
Bug (UI): Extra word in the description of the error

### ISSUE THREAD

**Issue by AnastasiaVolk** [getAlby/lightning-browser-extension#1137]
### Bug description

There is an extra word in the description of the error when using the wrong password to unlock the Alby Extension

 ### Screenshot
<img width="463" alt="Unlock to continue" src="https://user-images.githubusercontent.com/108762093/178320886-d5a8a8bd-d950-4fa6-bc27-045d3cfdbd55.png">

### To Reproduce

1. Press Alby extension button
2. “Unlock to continue” window appears
3. Input incorrect password 
4. “Invalid password (config)” noticement appears

### Expected behavior

“Invalid password” noticement

### Alby information

Alby Version: [v1.11.2]
Alby installed through Google Chrome

### Device information [optional]:

OS: [macOS Monterey 12.0.1 ] 
Browser [chrome]  Browser Version [Chrome Version 103.0.5060.114]

### Are you working on this issue?

No

**Comment by escapedcat** [comment]
Funny. Looks like it has been added on purpose:
https://github.com/getAlby/lightning-browser-extension/pull/478/files#diff-f247f31b0ca73e2fd79cfa8c84593a2fdd8e7e49ac46de7ba6470e5e6c883c90R62-R68
The word "config" is clickable and leads you to the welcome screen so you could re-add your wallet and set a new password.
But no user will understand this :)

We could i.e.:
- Reword it to "Re-add my account"
- Instead of a clickable string make it i.e. a button or a link so the user sees it's clickable

@AnastasiaVolk @bumi @dvoroneca  what do you think?

**Comment by AnastasiaVolk** [comment]
@escapedcat yeap, for sure the user needs to see that it is clickable, then it will make sense! And maybe reword as well)

**Comment by dvoroneca** [comment]
I've also seen this and it is addressed in the pop up log in variations in designs

**Comment by escapedcat** [comment]
Let's go with this for now if someone wants to tackle this:


>  - Reword it to "Re-add my account"
>  - Instead of a clickable string make it i.e. a button or a link so the user sees it's clickable

Just a link will probably be easier.

If nobody grabs this we will tackle this with the new design later on.

**Comment by bumi** [comment]
This is basically there since the first lines of Alby. It never was changed because not enough people complained about it and because it is a bit more tricky to get it right:

The user sees this when they forgot the password. Then the question is: what to do? There is no password recovery because all data is stored locally, encrypted.
Should this link completely reset Alby? Should it delete the stored, encrypted account? 

We can change the text there, but the bigger problem is to define the flow when a user forgot the password.

**Comment by bumi** [comment]
Here is a suggestion: 

in the case the user lost the password:
We warn the user, then delete the user's accounts and link the user again to the onboarding.

As text we use a link "Forgot password" that opens a modal with explanation and then the reset button. 

what do you think?

@pooniajay can write an entry in the guide about the Lost Password situation to which we link to?

**Comment by AnastasiaVolk** [comment]
Cool. That would be a new script for test case as well (added on Wiki)

**Comment by bumi** [comment]
we now have a different message and allow the user to reset their accoutns
